packages/@react-aria/table/stories/example.tsx

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {mergeProps} from '@react-aria/utils';
+import React from 'react';
+import {useCheckbox} from '@react-aria/checkbox';
+import {useFocusRing} from '@react-aria/focus';
+import {useRef} from 'react';
+import {useTable, useTableCell, useTableColumnHeader, useTableHeaderRow, useTableRow, useTableRowGroup, useTableSelectAllCheckbox, useTableSelectionCheckbox} from '@react-aria/table';
+import {useTableState} from '@react-stately/table';
+import {useToggleState} from '@react-stately/toggle';
+import {VisuallyHidden} from '@react-aria/visually-hidden';
+
+export function Table(props) {
+  let state = useTableState({...props, showSelectionCheckboxes: props.selectionMode === 'multiple'});
+  let ref = useRef();
+  let bodyRef = useRef();
+  let {collection} = state;
+  let {gridProps} = useTable({...props, scrollRef: bodyRef}, state, ref);
+
+  return (
+    <table {...gridProps} ref={ref} style={{borderCollapse: 'collapse'}}>
+      <TableRowGroup type="thead" style={{borderBottom: '2px solid gray', display: 'block'}}>
+        {collection.headerRows.map(headerRow => (
+          <TableHeaderRow key={headerRow.key} item={headerRow} state={state}>
+            {[...headerRow.childNodes].map(column =>
+              column.props.isSelectionCell
+                ? <TableSelectAllCell key={column.key} column={column} state={state} />
+                : <TableColumnHeader key={column.key} column={column} state={state} />
+            )}
+          </TableHeaderRow>
+        ))}
+      </TableRowGroup>
+      <TableRowGroup ref={bodyRef} type="tbody" style={{display: 'block', overflow: 'auto', maxHeight: '200px'}}>
```

Comment:
The same code from the usetable aria example except for the changes to make the table body scrollable and changes to work with the useSelectableCollection changes